### DISCUSSION TITLE
Transferring a save file from Sword to Shield

### DISCUSSION THREAD

**Discussion by javoerro** [kwsch/PKHeX#4381] [General]
Hello!

I'm trying to transfer a save file from Sword to Shield to avoid having to start a new game from scratch. Here’s what I did:

    -I used JKSV to extract the original Sword save file.
    -I loaded it into the editor, went to the SAV > Trainer Info tab, and switched the game version from Sword to Shield in the dropdown menu.
    -Then, I injected the modified save file into the Shield save directory using JKSV.

The original progress was just after defeating the Rock-type Gym, only two badges away from the League. However, I noticed that, in Shield game, **the Gym buildings in the overworld map for Fighting and Rock types were still present instead of the expected Ghost and Ice types**.

I was able to progress without major issues until the quest to catch Zamazenta. At that point, **I was supposed to return to the Ghost-type Gym, but because the Fighting-type Gym was still there instead, entering it causes the game to crash due to an unexpected error**.

It seems that certain in-game data tied to optional stadiums depends on specific save data. Is there any known way to adjust these data to resolve the issue and allow the quest to continue properly? Or any alternative methods for transferring a save from Sword to Shield and viceversa?

This would be a very useful feature. I've successfully done similar transfers in previous generations, and it’s a great way to access version-exclusive Pokémon in a more "legit" way.

Thanks so much for your help!

**Comment by Lusamine** [comment]
Research needs to be done to know what has to be changed for this kind of feature to be added. That means research into which parts of the save file are different between games (there are going to be lots of them due to things like spawn, trainer cards, gyms, vanish flags, etc). Until then, it's expected behavior that you will have irregularities if you swap from one game to another.

Since this is essentially a request for research, I'll convert this to a discussion where people can contribute if they want to.

**Comment by kwsch** [comment]
You'll spend more time trying to figure out how to correctly do this (i.e. figuring out what blocks need to change, and writing the code to do 100% of the work) compared to just playing the game again on the other version. 

It's not worth the effort or interest.
